test/TensorFlowRuntime/python_conversion.swift

```diff
@@ -4,12 +4,12 @@
 // REQUIRES: swift_test_mode_optimize
 // REQUIRES: tensorflow
 //
-// `numpy.ndarray` conversion tests.
+// `numpy.ndarray` and Python conversion tests.
 
 import TensorFlow
 import StdlibUnittest
 
-var NumpyConversionTests = TestSuite("NumpyConversion")
+var PythonConversionTests = TestSuite("PythonConversion")
```

Comment:
I believe you'll need to rename usages of `NumpyConversionTests` to `PythonConversionTests` as well.